template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/feature_engineering/notebooks/GenerateAndWriteFeatures.py.tmpl

```diff
@@ -103,7 +103,7 @@ raw_data = spark.read.format("delta").load(input_table_path)
 # Compute the features. This is done by dynamically loading the features module.
 from importlib import import_module
 
-mod = import_module(features_module)
+mod = import_module(features_module) ##??
```

Comment:
What is this comment for?

Reply:
> What is this comment for?

Good catch! Should have been omitted -- maybe I deleted after the commit -- will recommit